vulture/core.py

```diff
@@ -231,25 +232,29 @@ def by_size(item):
         def by_name(item):
             return (item.filename.lower(), item.lineno)
 
-        return sorted(
-            self.unused_attrs + self.unused_classes + self.unused_funcs +
-            self.unused_imports + self.unused_props + self.unused_vars,
-            key=by_size if self.sort_by_size else by_name)
+        unused_code = (self.unused_attrs + self.unused_classes +
+                       self.unused_funcs + self.unused_imports +
+                       self.unused_props + self.unused_vars)
+        unreachable_code = self.code_after_return
+
+        return sorted(unused_code + unreachable_code,
+                      key=by_size if self.sort_by_size else by_name)
 
     def report(self):
         """
         Print ordered list of Item objects to stdout.
         """
-        for item in self.get_unused_code():
+        for item in self.get_dead_code():
             if self.sort_by_size:
                 line_format = 'line' if item.size == 1 else 'lines'
                 size_report = ' ({0:d} {1})'.format(item.size, line_format)
             else:
                 size_report = ''
-
-            print("{0}:{1:d}: Unused {2} '{3}'{4}".format(
+            name_report = ("'{0}'".format(item.name) if item.name != ''
+                           else "")
+            print("{0}:{1:d}: Unused {2} {3}{4}".format(
                 utils.format_path(item.filename), item.lineno, item.typ,
-                item.name, size_report))
+                name_report, size_report))
```

Comment:
I was quite unclear as to what should be printed when reporting unreachable code, so decided to print nothing instead. Does it seems sound?